packages/datadog-plugin-jest/src/index.js

```diff
@@ -58,6 +64,89 @@ class JestPlugin extends Plugin {
     this.testEnvironmentMetadata = getTestEnvironmentMetadata('jest', this.config)
     this.codeOwnersEntries = getCodeOwnersFileEntries()
 
+    this.addSub('ci:jest:session:start', (command) => {
+      if (!this.config.isAgentlessEnabled) {
```

Comment:
Why is this needed for the suite spans but not the test spans?

Reply:
session and suite events are only processed by the agentless intake. Tests are processed by both agent and agentless intake